lib/grpc/server/adapters/cowboy.ex

```diff
@@ -203,7 +203,8 @@ defmodule GRPC.Server.Adapters.Cowboy do
           # https://github.com/ninenines/cowboy/issues/1398
           # If there are 1000 streams in one connection, then 1000/s frames per stream.
           max_received_frame_rate: {10_000_000, 10_000},
-          max_reset_stream_rate: {10_000, 10_000}
+          max_reset_stream_rate: {10_000, 10_000},
+          stream_window_update_threshold: 65_500
```

Comment:
Why do we have to change the cowboy adapter here?

Reply:
We don't need. I'll remove this. I was testing something and probably commit the change. M'bad